src/style/BookAdd.css

```diff
@@ -0,0 +1,38 @@
+@import url('https://fonts.googleapis.com/css2?family=Roboto+Slab&display=swap');
+
+.book__addition {
+  margin: 0 6.25rem;
+}
+
+.form__input {
+  width: 25%;
+  background-color: white;
```

Comment:
- [ ] It would be better if we implement the CSS DRY principle, these rules have been repeated. Let us kindly fix them